go/pkg/client/tree_whitebox_test.go

```diff
@@ -0,0 +1,52 @@
+package client
+
+import "testing"
+
+func TestGetTargetRelPath(t *testing.T) {
+	execRoot := "/execRoot/dir"
+	symlink := "sym"
+	tests := []struct {
+		desc      string
+		target    string
+		isError   bool
```

Comment:
nit: I think `wantErr` is more common in Go.

Reply:
i see, done